packages/client/README

```diff
@@ -0,0 +1,72 @@
+# Running Tests
+
+This project includes unit tests and end-to-end (E2E) tests. Below are the instructions to run the E2E tests.
+
+## Steps to Run E2E Tests
+
+1. In the terminal, navigate to the contracts directory:
+````cmd
+    cd packages/contracts && forge build
```

Comment:
if you add a build script on `contracts/package.json` it could be called like so

```suggestion
    yarn contracts build
```